package.json

```diff
@@ -53,7 +53,9 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next build && next start",
+    "start": "next start",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
```

Comment:
react-scripts does not exist in next.js and needs to be removed

Reply:
Great catch, I accidentally introduced them in a rebase conflict.